tests/interface/TWHDWalletTests.cpp

```diff
@@ -91,10 +91,11 @@ TEST(HDWallet, DeriveBitcoin) {
 TEST(HDWallet, DeriveTezos) {
     auto wallet = WRAP(TWHDWallet, TWHDWalletCreateWithMnemonic(words.get(), passphrase.get()));
     auto key = WRAP(TWPrivateKey, TWHDWalletGetKey(wallet.get(), TWCoinTypeTezos, 0, 0, 0));
-    auto publicKey = TWPrivateKeyGetPublicKeySecp256k1(key.get(), false);
+    auto publicKey = TWPrivateKeyGetPublicKeyEd25519(key.get());
     auto publicKeyData = WRAPD(TWPublicKeyData(publicKey));
```

Comment:
Tezos should be using `Secp256k1` ?

Reply:
this test is failing, I can remove these changes until i get it working right.

Reply:
I am going to revert this file.

If there is a bug in how we handle addresses, lets address in a separate PR and try to keep this PR minimally focused.

I'll follow up later today about proper address derivation.